addons/friendly_tracker/functions/fnc_init.sqf

```diff
@@ -0,0 +1,30 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3, veteran29
+ * Handles initialization of BFT loop
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call afm_friendly_tracker_fnc_init
+ *
+ * Public: No
+ */
+
+if (!hasInterface) exitWith {};
+
+if (isNil(QGVAR(running))) then {GVAR(running) = false};
+
+[{!GVAR(running)}, {
+	GVAR(running) = true;
+
+	LOG("Starting FriendlyTracker");
+
+	GVAR(markers) = [];
+
+    [QGVAR(loop)] call CBA_fnc_localEvent;
+}, [], GVAR(refreshRate) + 1, {}] call CBA_fnc_waitUntilAndExecute;
```

Comment:
With globals initialization in `preInit` the whole function could be simplified to something like:
```sqf
if (!hasInterface || {GVAR(running)}) exitWith {};

LOG("Starting FriendlyTracker");

GVAR(running) = true;

[QGVAR(loop)] call CBA_fnc_localEvent;

nil
```